common/env.go

```diff
@@ -23,9 +24,15 @@ func GetEnviroment() *Enviroment {
 	appHost := os.Getenv("APPLICATION_HOST")
 	appPort := os.Getenv("APPLICATION_PORT")
 	routerUrl := appHost + appPort
+	
+	var debug bool
+	if os.Getenv("LOGGER") == "true" {
```

Comment:
Env'i logger olarak değil DEBUG olarak al. sadece logger için bu ayarı yapmak zorunda değilsin belki başka paketlerini de debug modunda çalıştırmak istersin